app/assets/javascripts/search-vocab.js

```diff
@@ -0,0 +1,55 @@
+$(document).ready(function() {
+  $('.add-to-vocab-btn').click(function(e) {
+    e.preventDefault();
+    var signId = $(this).attr('data-sign-id');
+
+    var successNotice =
```

Comment:
How would you feel about having these elements always present in the sidebar but hidden and just toggling them to show/hide on success/failure?

Reply:
I kind of prefer this way now that we're using Ajax, so that if some one double-clicks, they get two messages. Thoughts?

Reply:
This is resolved by the current update :)